package.json

```diff
@@ -139,6 +139,7 @@
     "nock": "^13.2.2",
     "sass": "^1.63.6",
     "vite": "^4.3.9",
-    "vitest": "^0.34.6"
+    "vitest": "^0.34.6",
+    "@vitest/coverage-v8": "^0.34.6"
```

Comment:
Als ik deze installeer met `npm install -d @vitest/coverage-v8` komt hij voor jsdom in het lijstje te staan... wat zijn dat voor bewerkelijke tools waarmee je dit installeert?